app/decorators/physical_chassis_decorator.rb

```diff
@@ -0,0 +1,39 @@
+class PhysicalChassisDecorator < MiqDecorator
+  def self.fonticon
+    'pficon pficon-cluster'
+  end
+
+  def fileicon
+    "svg/vendor-#{vendor.downcase}.svg"
+  end
+
+  def quadicon
+    {
+      :top_left     => {
+        :text    => physical_servers.count,
+        :tooltip => _('Servers')
+      },
+      :top_right    => {
+        :fileicon => fileicon,
+        :tooltip  => ui_lookup(:model => type)
+      },
+      :bottom_left  => {
+        :fonticon => fonticon,
+        :tooltip  => ui_lookup(:model => type),
+      },
+      :bottom_right => {
+        :fileicon => health_state_img,
+        :tooltip  => _(health_state),
+      }
+    }
+  end
+
+  def health_state_img
```

Comment:
You no longer need this, rebase to the latest master and you can use the `QuadiconHelper.provider_status`, for inspiration check the other physical decorators, they have it already.